pydatalab/src/pydatalab/models/__init__.py

```diff
@@ -25,3 +25,9 @@
     "Equipment",
     "ITEM_MODELS",
 )
+
+Sample.model_rebuild()
+StartingMaterial.model_rebuild()
+Cell.model_rebuild()
+Equipment.model_rebuild()
+Collection.model_rebuild()
```

Comment:
```suggestion
{model.model_rebuild() for model in ITEM_MODELS.values()}
```

Might be neater, though not sure why we need this now and not before (will play around with it later)

Reply:
Doesn't work with pre-commit for me, need to find out why.

It seems that this is necessary because Pydantic v2 is stricter with forward reference and circular dependencies

Reply:
Also need to keep Collection since it's not in ITEM_MODELS